src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -40,7 +40,10 @@ case class Config(
                  shouldViperEncode: Boolean = true,
                  checkOverflows: Boolean = false,
                  shouldVerify: Boolean = true,
-                 int64bit: Boolean = false // picks the size of int's, 32 if false, 64 bit otherwise
+                 // The go language specification states that int and uint variables can have either 32bit or 64, as long
+                 // as they have the same size. This flag allows users to pick the size of int's and uints's: 32 if false,
+                 // 64 bit otherwise.
+                 int64bit: Boolean = false
```

Comment:
From my cursory glance at the spec I understand it to mean that the size of `int`s and `uint`s is actually based on the CPU architecture/word size. Would it be reasonable to try to detect this at runtime?

As a separate point, would a 64-bit default not be better, given how common 64-bit architectures are in personal computers now?

Reply:
I would not try to detect this at runtime (at least, not by default) because you may be performing verification on a different machine than the one that will run the code (e.g. in case you want to perform verification on a more powerful machine). I agree with your second point

Reply:
@jcp19 I agree with you as well. Will you adapt the default value of `int64bit`?

Reply:
Yes, changed default int size to 64bit